nativelib/src/main/scala/scala/scalanative/runtime/time.scala

```diff
@@ -5,10 +5,23 @@ import scala.scalanative.unsafe.{CLongLong, extern}
 
 @extern
 object time {
-  def scalanative_nano_time: CLongLong = extern
-  def scalanative_current_time_millis: CLongLong = extern
 
-  /** Time zone offset in seconds
+  /** Monotonically increasing time for use in timers and for
+   *  [[java.lang.System$.nanoTime()* System.nanoTime()]] in the Java library.
+   *  @return
+   *    increasing time hopefully at better than millisecond resolution
+   */
+  def scalanative_nano_time(): CLongLong = extern
+
+  /** Milliseconds from the UNIX epoch to implement
+   *  [[java.lang.System$.currentTimeMillis()* System.currentTimeMillis()]] in
+   *  the Java library.
+   *  @return
+   *    time in millis (UTC)
+   */
+  def scalanative_current_time_millis(): CLongLong = extern
+
+  /** Time zone offset in seconds, designed to be used by [[java.util.Date]]
```

Comment:
Not to offend by mentioning it again.  It is neigh on impossible to stamp put confusion about time zone offsets. On can however reduce confusion by specifying which of the two prominent conventions which is used.  

That is, "Time zone offset in seconds" would be easier for people reading on the fly to understand if it were "Time zone offset in seconds East of UTC".

Someday the generations which expect "West of UTC" will retire, die off, or learn to be quiet.

Reply:
I changed the comment hopefully so anyone can understand without knowing any history.